src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -2939,6 +2938,11 @@ private static bool AttributeTypeNamesMatch(Type attributeType, Type reflectedAt
                     }
                 }
 
+                // Scoping the call to GetFields to a local function to limit the linker suppression
+                [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2070:UnrecognizedReflectionPattern",
+                Justification = "Nested type members are safe from linker with the parent annotation")]
```

Comment:
(nit) The `Justification` line should be indented one level to the right.

(nit) "safe from linker" sounds a bit like the linker is dangerous. How about "Nested type members will be preserved by the ILLinker with the parent annotation" ?

Reply:
Or just skip the "linker" part altogether. "Nested type members will be preserved with the parent annotation"

.NET is trying to overload what the term [linker](https://en.wikipedia.org/wiki/Linker_(computing)) means in computing. That is never a good idea. IL Linker is not doing any of what anyone outside .NET thinks of when they hear "linker". We should stay away from terms linking/linker as much as possible and use the term "trimming".

We are introducing unnecessary confusion into .NET with naming the tool "linker". It's hard to go back on the naming there, but we can make conscious effort to not have the name leak into other parts of .NET. The user-visible parts of project settings already do a pretty good job of avoiding the unfortunate name.